source/includes/aggregation/atlas-vector-search-example.js

```diff
@@ -0,0 +1,49 @@
+const { MongoClient } = require("mongodb");
+
+// Replace with the connection string to your Atlas cluster to connect
+const uri = "<connection-string>";
+
+const client = new MongoClient(uri);
+
+async function run() {
+    try {
+        await client.connect();
+
+        // Sets the namespace
+        const database = client.db("sample_mflix");
+        const coll = database.collection("embedded_movies");
+
+        // Defines the pipeline
+        const agg = [
+            {
+              '$vectorSearch': {
+                'index': 'vector_index',
+                'path': 'plot_embedding',
+                'queryVector': [
```

Comment:
I think this should be using the Vector API for a more efficient data type when serializing to BSON:

```js
const { MongoClient, Binary } = require("mongodb");
// ...
'queryVector': Binary.fromFloat32Array(Float32Array.from([...]))
```